site/docs/label.md

```diff
@@ -1,8 +1,86 @@
 ---
 layout: docs
 menu: docs
-title: Labeling (Coming Soon!)
+title: Labeling
 permalink: /docs/label.html
 ---
 
-Labeling will be supported soon. Please subscribe to [our repository](https://github.com/vega/vega-lite) to hear about our releases.
+Labels can provide details of a particular data point as a text annotation directly near the data point. Users can add text labels to marks of the following types: area, bar, line, trail, rect, circle, point, and square. To annotate marks with labels, users can encode fields with label encoding channel.
+
+<!--prettier-ignore-start-->
+## Documentation Overview
+{:.no_toc}
+
+- TOC
+{:toc}
+
+<!--prettier-ignore-end-->
+
+## Syntax
+
+```js
+{
+  ...,
+  mark: ...,
+  encoding: {
+    x: {...},
+    y: {...},
+    label: {
+      // field(s) to encode as text label
+      field: ...,
+
+      // label properties
+      positions: ...,
+      avoid: ...,
+      mark: ...,
+      padding: ...,
+      method: ...,
+      lineAnchor: ...,
+    }
+  }
+}
+```
+
+## Configurations
```

Comment:
Are these configurations as in theme?  We use the term "properties" in other pages.

Reply:
Also, you need to define that this table lists properties of label field definition, in addition to standard  field definition property. 

See the language we use in encoding.md (https://vega.github.io/vega-lite/docs/encoding.html#text-field-def). 

- [ ] Please also add a "label" to the "text and tooltip" section in the encoding.md 
- [ ] rename the section to "text, label, and tooltip" 
- [ ] link to this page, just like how tooltip links to its own page.